Assets/AnkrSDK/Scripts/Mobile/MobileWalletHandler.cs

```diff
@@ -1,19 +1,30 @@
+using System.Collections.Generic;
 using AnkrSDK.Core.Infrastructure;
 using AnkrSDK.Utils;
 using AnkrSDK.WalletConnectSharp.Unity;
 using Cysharp.Threading.Tasks;
 
 namespace AnkrSDK.Mobile
 {
-	public class MobileDisconnectHandler : IDisconnectHandler
+	public class MobileWalletHandler : IWalletHandler
 	{
 		private readonly WalletConnect _walletConnect;
-
-		public MobileDisconnectHandler()
+ 
+		public MobileWalletHandler()
 		{
 			_walletConnect = ConnectProvider<WalletConnect>.GetWalletConnect();
 		}
 
+		public UniTask<Dictionary<string, bool>> GetWalletsStatus()
```

Comment:
Create a named class for <string, bool> tuple

Reply:
Put a dictionary inside that class. Something like
WalletsStatusData{
public Dictionary< WalletStatusData >

}
WalletStatusData{
public string
public bool
}

Reply:
As I know Dictionaty need key and value types. In that case we have only way like that
`public class WalletStatusData : Dictionary<string, bool> {}`
or I need to change JS lib